assets/templateFloxEnv/manifest.toml

```diff
@@ -2,6 +2,10 @@
 [install]
 # hello = {}
 
+[registry.inputs.nixpkgs]
+from = { type = "github", owner = "NixOS", repo = "nixpkgs", rev = "e8039594435c68eb4f780f3e9bf3972a7399c4b1" }
```

Comment:
suggestion blocking: can we leave this unlocked in the template?

Reply:
This should not appear in the template at all after @garbas completes his work; but if you do include it for now - do not pin the revision.

If you need pinned revisions for tests as an optimization, handle that separately.